sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -745,6 +759,13 @@
     "image_classification_job = automl.image_classification(\n",
     "    compute=compute_name,\n",
     "    experiment_name=exp_name,\n",
+    "    properties={\n",
+    "\n",
```

Comment:
@KonakanchiSwathi - Please revert the changes used for testing purroses.

Reply:
removed